src/lib/shady-render.ts

```diff
@@ -157,12 +158,20 @@ const prepareTemplateStyles =
       }
     };
 
+export interface ShadyRenderOptions extends Partial<RenderOptions> {
+  scopeName: string;
+}
+
 export function render(
     result: TemplateResult,
     container: Element|DocumentFragment,
-    scopeName: string) {
+    options: ShadyRenderOptions) {
+  const scopeName = options.scopeName;
   const hasRendered = parts.has(container);
-  litRender(result, container, shadyTemplateFactory(scopeName));
+  litRender(result, container, {
+    eventContext: options.eventContext,
+    templateFactory: shadyTemplateFactory(scopeName),
+  });
```

Comment:
It seems footgunny to have each `render` need to know about and forward on all the options in existence (TypeScript won't help remind you when you forgot to add a new option here if they're optional, right? or did I miss something?)... could we have the pattern be:
```js
litRender(result, container, {...options, templateFactory: shadyTemplateFactory(options.scopeName)});
```
Is the answer "no", because you need to filter out the extra options the the base render doesn't define to have the type work?